python/ray/__init__.py

```diff
@@ -155,6 +155,7 @@ def _configure_system():
 # some functions in the worker.
 import ray.actor  # noqa: E402,F401
 from ray.actor import method  # noqa: E402
+from ray.cross_language import java_function, java_actor_class  # noqa: E402
```

Comment:
Can you add a `TODO` here which indicates that we should remove this in 2.0?

Reply:
Let us see if there is a tracker for 2.0 API changes. If yes, it's no need to add TODO right?

Reply:
@SongGuyang  I added a todo comment and left it in my private TODO list.